src/lib/chat/matrix-client.ts

```diff
@@ -396,14 +396,20 @@ export class MatrixClient implements IChatClient {
     const options: ICreateRoomOpts = {
       preset: Preset.TrustedPrivateChat,
       visibility: Visibility.Private,
-      invite: users.map((u) => u.matrixId),
+      invite: [],
       is_direct: true,
       initial_state,
       power_level_content_override: {
         users: {
           ...users.reduce((acc, u) => ({ ...acc, [u.matrixId]: PowerLevels.Viewer }), {}),
```

Comment:
Can remove this now?

Reply:
i left it in there as a "just in case"/"good to have". But yeah we can remove now. will do.